ezar/cogs/misc.py

```diff
@@ -0,0 +1,66 @@
+from typing import Literal
+
+from disnake import CommandInter, Member, User
+from disnake.ext.commands import Cog, slash_command
+
+from ezar import Ezar
+from ezar.backend.config import Colors
+from ezar.utils.embed import Embeb
+
+
+class Miscellaneous(Cog):
+    """Miscellaneous/not-important commands"""
+
+    def __init__(self, bot: Ezar):
+        self.bot = bot
+
+    @slash_command()
+    async def icon(self, itr: CommandInter):
+        """Icon/Profile-Picture-related commands"""
+
+    @icon.sub_command()
+    async def server(self, itr: CommandInter):
+        """Returns the server icon URL."""
+        avatar_embed = Embeb()
+        avatar_embed.set_image(url=itr.guild.icon.with_size(1024).url)
+        avatar_embed.set_author(
+            name=itr.guild.name,
+            icon_url=itr.guild.icon if itr.guild.icon else itr.guild.gen,
+        )
+        await itr.response.send_message(embed=avatar_embed)
+
+    @icon.sub_command()
+    async def user(
+        self,
+        itr: CommandInter,
+        user: User = None,
```

Comment:
```suggestion
        user: Optional[User] = None,
```
This would require an import, but as you use vscode, I would recommend setting
```toml
[tool.pyright]
typeCheckingMode = "basic"
```
in `pyproject.toml`, and/or `"python.analysis.typeCheckingMode": "basic",` in vscode settings for actual type checking if you wish (Maybe a github check? but that might be a bit far for now).

Reply:
A GitHub check may be alright, but yes, I suppose it's alright for now, but thank you.